### ISSUE TITLE
UTF ANSI conversion

### ISSUE THREAD

**Issue by SimonSteinberger** [Linus789/dictcc-stardict#1]
The default file encoding by dict.cc is UTF-8, but the converter seems to need ANSI encoding. I saved the dict.cc fle in ANSI and it worked perfectly. This could be done automatically by the script - or you could mention this pitfall in the readme file.

Thanks so much for this tool!

**Comment by Linus789** [comment]
Hmm, what exactly went wrong when you didn't convert the file beforehand? Did you use Python 3? If yes, does it help if you replace the two instances of
```python
open(input_file)
```
with
```python
open(input_file, encoding="utf-8")
```
?

**Comment by SimonSteinberger** [comment]
Nice, that does the trick 👍  I replaced both instances `open(...)` and it works.

Before, I got this error:

```
Traceback (most recent call last):
  File "D:\Temp\convert.py", line 410, in <module>
    main(args.file, args.source_lang)
  File "D:\Temp\convert.py", line 266, in main
    lang_pair = get_language_pair(input_file)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Temp\convert.py", line 15, in get_language_pair
    return f.readline().removeprefix("#").strip().split()[0].lower()
           ^^^^^^^^^^^^
  File "D:\Web\Django\Python3\Lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 1191: character maps to <undefined>
Traceback locals:
    self = <encodings.cp1252.IncrementalDecoder object at 0x000001E4DAD76310>
    input = b"# EN-DE vocabulary database\tcompiled by dict.cc\n# Date and ti...
    len(input) = 8192
    final = False

```

**Comment by Linus789** [comment]
Now fixed with commit 229db31. Thanks for your bug report!
